bedrock/firefox/templates/firefox/new/batm/machine.html

```diff
@@ -0,0 +1,45 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% extends "firefox/new/batm/base.html" %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_new_batm_machine' %}
+{% endblock %}
+
+{% block body_class %}scene-1 machine-animated{% endblock %}
+
+{% block content %}
+<main role="main">
+  <section class="machine">
+    <video id="machine" preload="auto" autoplay loop muted>
+      <source src="https://assets.mozilla.net/video/batm-animation.webm" type="video/webm">
+      <source src="https://assets.mozilla.net/video/batm-animation.mp4" type="video/mp4">
+    </video>
+  </section>
+
+  <div class="cta-copy">
+    <h1 id="main-header-copy" data-experience="batmprivate" data-variant="b">{{_('Browse against the machine')}}</h1>
+    <p class="tagline">
+      {{ _('Firefox doesn’t sell access to your personal information.') }}
+      {{ _('From privacy tools to tracking protection, you control who sees what.') }}
+    </p>
+  </div>
+
+  <div class="cta-download">
+    {{ download_firefox(alt_copy=_('Download Firefox'), locale_in_transition=True, button_color='button-arrow') }}
+  </div>
+
+</main>
+{% endblock %}
+
+{% block js %}
+  <!--[if IE 9]>
+    {% javascript 'matchmedia_addlistener' %}
+  <![endif]-->
```

Comment:
Do we still need this lib included? Looks like all the `matchMedia` code was removed from the JS file...